src/pages/index.js

```diff
@@ -1,211 +1,253 @@
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "298fcaaa-e173-4e81-9326-ea95e21a4625",
+    "Content-Type": "application/json",
+  },
+});
+
+// Instantiate user info
+const user = new UserInfo({
+  title: ".profile__title",
+  subtitle: ".profile__subtitle",
+  avatar: ".profile__image",
+});
+
+// Instantiate card section
+export const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (item) => {
-      section.addItem(createCard(item));
+      const cardElement = createCard(
+        item,
+        handleCardImageClick,
+        handleDeleteClick,
+        handleLikeClick
+      );
+      cardSection.addItem(cardElement);
     },
   },
   "#card-list"
 );
-section.renderItems();
 
-// User Info
-const user = new UserInfo({
-  title: ".profile__title",
-  subtitle: ".profile__subtitle",
-});
+// ! INITIAL DATA FETCHING
+
+// Fetch initial cards and user info
+const initialize = async () => {
+  try {
+    const [cards, userInfo] = await Promise.all([
+      api.getInitialCards(),
+      api.getUserInfo(),
+    ]);
+
+    cardSection.setItems(cards);
+    user.setUserInfo({
+      name: userInfo.name,
+      about: userInfo.about,
+    });
+    user.setUserAvatar({
+      avatar: userInfo.avatar,
+    });
+  } catch (err) {
+    console.error("Initialization error:", err);
+  }
+};
+
+initialize();
+
+// ! EVENT HANDLERS
+
+// Handle profile edit form submission
+function handleProfileEditSubmit(profileData) {
+  return handleSubmit(() => {
+    return api
+      .updateProfileInfo({
+        name: profileData.title,
+        description: profileData.subtitle,
+      })
+      .then((updatedData) => {
+        user.setUserInfo(updatedData);
+      });
+  }, profileEditPopup);
+}
+
+// Add card handler
+function handleAddCardSubmit(inputValues) {
+  return handleSubmit(() => {
+    return api
+      .addCard({ name: inputValues.title, link: inputValues.image })
+      .then((newCardData) => {
+        // Use the createCard utility to create a new card
+        const newCardElement = createCard(
+          newCardData,
+          handleCardImageClick,
+          handleDeleteClick,
+          handleLikeClick
+        );
+
+        // Add the new card to the card section
+        cardSection.addItem(newCardElement);
+      });
+  }, addCardPopup);
+}
 
-// ! EVENT LISTENERS FOR POPUPS
+// Handle profile image form submission
+function handleProfileImageSubmit(newImageData) {
+  return handleSubmit(() => {
+    return api
+      .updateProfileAvatar({ avatar: newImageData.image })
+      .then((res) => {
+        user.setUserAvatar({ avatar: res.avatar });
+      });
+  }, profileImagePopup);
+}
+
+// Handle card delete confirmation
+function handleDeleteClick(card) {
+  cardToDelete = card;
+  deleteCardPopup.open();
+}
 
+// Handle card like button
+function handleLikeClick(card) {
+  const newLikeState = !card._isLiked;
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.